com.protonvpn.www.yml

```diff
@@ -13,6 +13,9 @@ finish-args:
   - --share=network
   - --socket=wayland
   - --socket=fallback-x11
+  # AT-SPI accessibility bus
+  - --talk-name=org.a11y.Bus
+  - --filesystem=xdg-run/at-spi
```

Comment:
Why are you doing this?

Flatpak sets the `AT_SPI_BUS_ADDRESS` env var and all applications/libraries should be respecting it.

Reply:
And this is a gtk3 app, it *should* just work.